app/routes/ilpay.tsx

```diff
@@ -39,7 +39,7 @@ export default function Ilpay() {
         <div className="h-2/3 items-center justify-center flex flex-col gap-10 w-full max-w-sm">
           <DialPad />
           <div className="flex gap-2">
-            <Link to={`/request`}>
+            <Link to={Number(amountValue) === 0 ? "#" : `/request`}>
```

Comment:
would `onClick={ (event) => event.preventDefault() }` be a better approach when amount is 0, so clicking the link would not do anything, rather then put a # in url ?